app/Http/Controllers/User/Dashboard/DashboardController.php

```diff
@@ -23,4 +23,10 @@ public function home()
         $user = Auth::user() ?? $this->user;
         return view('dashboard.home', compact('user'));
     }
+
```

Comment:
I see a Dashboard Controller under User and also under Admin. I don't believe there is need for a User Dashboard, at least with this story. Let me know if I am not getting it

Reply:
This was created to allow a normal user to login and show a forbidden message with a navbar to logout, but we can just not allow the user to login.